lsp-mode.el

```diff
@@ -3916,14 +3916,20 @@ WORKSPACE is the active workspace."
       ;; screwed up with a previous Content-Length
       (error "No Content-Length header"))))
 
+(defun s-trim-left (s)
```

Comment:
There is already such function(in `s` package), you could delete it.

Reply:
oh oops doh. removed